test/parallel/test-crypto-binary-default.js

```diff
@@ -61,353 +61,366 @@ assert.throws(function() {
+        sha256:
+            'b0344c61d8db38535ca8afceaf0bf12b881dc200c9833da726e9376c' +
+            '2e32cff7',
+        sha384:
+            'afd03944d84895626b0825f4ab46907f15f9dadbe4101ec682aa034c' +
+            '7cebc59cfaea9ea9076ede7f4af152e8b2fa9cb6',
+        sha512:
+            '87aa7cdea5ef619d4ff0b4241a1d6cb02379f4e2ce4ec2787ad0b305' +
+            '45e17cdedaa833b7d6b8a702038b274eaea3f4e4be9d914eeb61f170' +
+            '2e696c203a126854'
+      }
+    },
+    {
+      key: Buffer.from('4a656665', 'hex'), // 'Jefe'
+      data: Buffer.from('7768617420646f2079612077616e7420666f72206e6f74686' +
+                        '96e673f', 'hex'), // 'what do ya want for nothing?'
+      hmac: {
+        sha224: 'a30e01098bc6dbbf45690f3a7e9e6d0f8bbea2a39e6148008fd05e44',
+        sha256:
+            '5bdcc146bf60754e6a042426089575c75a003f089d2739839dec58b9' +
+            '64ec3843',
+        sha384:
+            'af45d2e376484031617f78d2b58a6b1b9c7ef464f5a01b47e42ec373' +
+            '6322445e8e2240ca5e69e2c78b3239ecfab21649',
+        sha512:
+            '164b7a7bfcf819e2e395fbe73b56e0a387bd64222e831fd610270cd7' +
+            'ea2505549758bf75c05a994a6d034f65f8f0e6fdcaeab1a34d4a6b4b' +
+            '636e070a38bce737'
+      }
+    },
+    {
+      key: Buffer.from('aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa', 'hex'),
+      data: Buffer.from('ddddddddddddddddddddddddddddddddddddddddddddddddd' +
+                        'ddddddddddddddddddddddddddddddddddddddddddddddddddd',
+                        'hex'),
+      hmac: {
+        sha224: '7fb3cb3588c6c1f6ffa9694d7d6ad2649365b0c1f65d69d1ec8333ea',
+        sha256:
+            '773ea91e36800e46854db8ebd09181a72959098b3ef8c122d9635514' +
+            'ced565fe',
+        sha384:
+            '88062608d3e6ad8a0aa2ace014c8a86f0aa635d947ac9febe83ef4e5' +
+            '5966144b2a5ab39dc13814b94e3ab6e101a34f27',
+        sha512:
+            'fa73b0089d56a284efb0f0756c890be9b1b5dbdd8ee81a3655f83e33' +
+            'b2279d39bf3e848279a722c806b485a47e67c807b946a337bee89426' +
+            '74278859e13292fb'
+      }
+    },
+    {
+      key: Buffer.from('0102030405060708090a0b0c0d0e0f10111213141516171819',
+                       'hex'),
+      data: Buffer.from('cdcdcdcdcdcdcdcdcdcdcdcdcdcdcdcdcdcdcdcdcdcdcdcdc' +
+                        'dcdcdcdcdcdcdcdcdcdcdcdcdcdcdcdcdcdcdcdcdcdcdcdcdcd',
+                        'hex'),
+      hmac: {
+        sha224: '6c11506874013cac6a2abc1bb382627cec6a90d86efc012de7afec5a',
+        sha256:
+            '82558a389a443c0ea4cc819899f2083a85f0faa3e578f8077a2e3ff4' +
+            '6729665b',
+        sha384:
+            '3e8a69b7783c25851933ab6290af6ca77a9981480850009cc5577c6e' +
+            '1f573b4e6801dd23c4a7d679ccf8a386c674cffb',
+        sha512:
+            'b0ba465637458c6990e5a8c5f61d4af7e576d97ff94b872de76f8050' +
+            '361ee3dba91ca5c11aa25eb4d679275cc5788063a5f19741120c4f2d' +
+            'e2adebeb10a298dd'
+      }
+    },
 
```

Comment:
This line seems obsolete and should probably be removed.

Reply:
Can you briefly explain why the line is obsolete?

Reply:
These are entries in an array and all other entries are separated without extra line break.

Reply:
@BridgeAR Fixed :)